src/parser/internal/classish.rs

```diff
@@ -90,6 +93,8 @@ impl Parser {
 
             self.lbrace(state)?;
 
+            let attributes = state.get_attributes();
```

Comment:
this is important, if we don't call `get_attributes` before parsing the body, the attributes might end up on a member instead of the trait/class/enum/interface.